From: Denis Auroux <auroux@users.sourceforge.net>
Date: Fri, 1 Jun 2018 10:33:38 -0700
Subject: upstream html install dir part 1

Install html documentation in docdir/htmldir instead of html-doc
---
 Makefile.am | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index 1c0d5b9..6c42573 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -18,14 +18,14 @@ install-data-local:
 	  done \
 	fi; \
 	if test -d $(srcdir)/html-doc; then \
-	  $(mkinstalldirs) $(DESTDIR)$(pkgdatadir)/html-doc; \
+	  $(mkinstalldirs) $(DESTDIR)$(htmldir); \
 	  for docfile in $(srcdir)/html-doc/*; do \
 	    if test -f $$docfile; then \
-	      $(INSTALL_DATA) $$docfile $(DESTDIR)$(pkgdatadir)/html-doc; \
+	      $(INSTALL_DATA) $$docfile $(DESTDIR)$(htmldir); \
 	    fi \
 	  done; \
-	  if test ! -e $(DESTDIR)$(pkgdatadir)/html-doc/pixmaps; then \
-	    ln -s ../pixmaps $(DESTDIR)$(pkgdatadir)/html-doc/pixmaps; \
+	  if test ! -e $(DESTDIR)$(htmldir)/pixmaps; then \
+	    ln -s $(DESTDIR)$(pkgdatadir)/pixmaps $(DESTDIR)$(htmldir)/pixmaps; \
 	  fi \
 	fi; \
 	echo "*** Desktop files, icons, MIME types not installed. Run 'make desktop-install'"; \
